src/apps/workspaces/crud/user_applet_access.py

```diff
@@ -671,20 +720,67 @@ async def get_workspace_managers(
             .where(
                 UserAppletAccessSchema.soft_exists(),
                 UserAppletAccessSchema.owner_id == owner_id,
-                UserAppletAccessSchema.role != Role.RESPONDENT,
                 has_access,
+                UserAppletAccessSchema.role != Role.RESPONDENT,
                 UserAppletAccessSchema.applet_id == applet_id if applet_id else True,
             )
-            .group_by(UserSchema.id, UserAppletAccessSchema.title)
+            .group_by(UserSchema.id)
+        ).cte("accepted_users")
+
+        query: Query = (
+            select(
+                func.coalesce(accepted_users.c.id, invited_users.c.id).label("id"),
+                func.coalesce(accepted_users.c.first_name, invited_users.c.first_name).label("first_name"),
+                func.coalesce(accepted_users.c.last_name, invited_users.c.last_name).label("last_name"),
+                func.coalesce(accepted_users.c.email_encrypted, invited_users.c.email).label("email_encrypted"),
+                func.coalesce(accepted_users.c.created_at, invited_users.c.created_at).label("created_at"),
```

Comment:
created_at is used in ordering

Reply:
Oh ok. It needs to be returned in the response as well because the FE will need it

Reply:
@rcmerlo referencing your comment from [here](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1403#discussion_r1638513877), I don't think these two dates are the same thing. What's the rationale behind making this decision on the BE?

Reply:
this is a list do managers and we need to merge manager and invitations in this one list with abstraction what each entry is. whats the diference to FE if this field is the date of user being created or the date invitation is create, it is the date of the creation of this entry. But FE can known it is a invitation because of status, so can show in label "invitation date"...  if status is pending and not show this date when approved. 

Please let's go forward with this ticket and tackle this in menu ticket

Reply:
I think you should run that by the design team before making that decision. They've made the call to show the invitation date after the team member has accepted the invitation, so we'll need to expose that unless they decide to make a change.

Reply:
This is not in the scope of this ticket, but we do not need an extra field, created_at will return what we need. I will rethink it in more menu ticket if needed. Please, unblock this PR